src/Metalama.Patterns.Observability/README.md

```diff
@@ -0,0 +1 @@
+This package is currently under development.
```

Comment:
Maybe add a link to the newsletter here, so that people can be notified when the package is released?

Reply:
Not important.